src/duplicacy_snapshot.go

```diff
@@ -76,19 +77,28 @@ func CreateSnapshotFromDirectory(id string, top string) (snapshot *Snapshot, ski
                 continue
             }
 
-            if pattern[0] != '+' && pattern[0] != '-' {
+            if pattern[0] == '#' {
+                continue
+            }
+
+            if pattern[0] != '+' && pattern[0] != '-' && pattern[0] != 'i' && pattern[0] != 'e' {
```

Comment:
From here down appears to duplicate the equivalent code in duplicacy_main.go. How about pulling out a helper method to avoid having to change things in multiple places in the future?

Reply:
done